src/su.c

```diff
@@ -313,6 +313,7 @@ static void prepare_pam_close_session (void)
 	}
 
 	/* parent only */
+	closelog();
```

Comment:
?

Reply:
Removed again. Sorry, had work in progress changes in tree.

Reply:
Yup, I guessed that.  :)